src/site/content/en/progressive-web-apps/progressive-web-apps.11tydata.js

```diff
@@ -10,12 +10,12 @@ module.exports = {
     title: "Progressive Web Apps",
     updated: "November 1, 2019",
     description: `Websites that took all the right vitamins`,
-    overview: `In this collection, you"ll learn what makes a Progressive Web
+    overview: `In this collection, you'll learn what makes a Progressive Web
```

Comment:
This overview doesn't actually tell me what a progressive web app is. I think it's possible to sum them up in a paragraph.